types/react-native/index.d.ts

```diff
@@ -3961,12 +3962,12 @@ export class ImageBackground extends ImageBackgroundBase {
     queryCache?(urls: string[]): Promise<{ [url: string]: 'memory' | 'disk' | 'disk/memory' }>;
 }
 
-export interface ViewToken {
-    item: any;
+export interface ViewToken<ItemT = unknown, SectionT = never> {
```

Comment:
Why is one `unknown` and the other `never`?

Reply:
Because this is a shared type between Flatlist and SectionList - in the case of a Flatlist, the property `section` is not actually provided at all at runtime, so in theory `never` is more correct IMO.

Reply:
Why can't we fork the type to accurately reflect the runtime value by not specifying `section` at all in that case?

Reply:
We can, however, the side effect of that is that we'll need to fork a few other types which all in turn use the view token type - I'm happy to do this if we are happy to introduce a bigger change.